harness/setup.py

```diff
@@ -14,19 +14,15 @@
     python_requires=">=3.6",
     package_data={"determined": ["py.typed"]},
     install_requires=[
-        "boto3>=1.9.220",
```

Comment:
were these not really needed?

Reply:
No, because they are already required by the common package.